src/main/resources/services/bkibol/bkibol.ts

```diff
@@ -16,13 +22,19 @@ function get(req: HttpRequestParams): XP.Response {
   const endMonth: string | undefined = req.params?.endMonth || ''
   const endYear: string | undefined = req.params?.endYear
   const language: string | undefined = req.params?.language ? req.params.language : 'nb'
+
+  const lang: Language = getLanguage(getContent()) as Language
```

Comment:
Do you need lang?, you already have language from req.params.language

Reply:
req.params.language is string, just nb etc while getLanguage gives object which contains phrases which is needed for allMonths